examples/complete/dev.tfvars

```diff
@@ -0,0 +1,5 @@
+secret_mapping = [{
+  name = "db_password"
+  file = "../../config/secrets/dev.yaml"
```

Comment:
This will blow up, yeah? Can we commit some fake secrets into the `examples/complete` so it will run?

Reply:
Will do it in the upcoming PR.